index.html

```diff
@@ -6,109 +6,132 @@
+                </svg>
+              </li>
+            </ul>
+          </div>
         </nav>
         <!-- SCROLL PROGRESS BAR -->
         <div class="scroll-tracker-backer"><div class="scroll-tracker"></div></div>        
       </header>
-      <!-- ANIMATED BACKGROUND IMAGE -->
-      <img class="background-svg rotate-in-y" src="assets/images/geometric_background_accent.svg" alt="Geometric background image">
-      <main>
+        <!-- ANIMATED BACKGROUND IMAGE -->
+        <div class="landing-background-wrapper">
+          <img class="masked" src="assets/images/hire-me.png" alt="Background image that is sure to get me hired">
+        </div>
+      <main class="content-wrapper">
         <!-- LANDING -->
-        <section id="landing" class="landing has-gutters">
+        <section id="landing" class="landing">
           <h1>Welcome.</h1>
           <div class="landing-content">
             <div class="landing-text">
               <p>Hello, my name is Andrew, and I am a web developer in training.</p>
             </div>
-            <a class="section-button grow" href="#about-me">About me</a>
-            <a class="section-button grow" href="#projects">Projects</a>
+            <a class="section-button change-color" href="#about-me">About me</a>
+            <a class="section-button change-color" href="#projects">Projects</a>
           </div>
         </section>
         <!-- ABOUT ME -->
-        <section class="about-me has-gutters" id="about-me">
-          <h1>About me.</h1>
-          <img id="profile-image" src="assets/images/Andy.png" alt="image of Andrew with a beard">
+        <section class="about-me" id="about-me">
+          <h2>About me.</h2>
+          <img class="profile-image" src="assets/images/Andy.png" alt="image of Andrew with a beard">
           <p>I am an analytical chemist with four years of experience in the field. I am currently transitioning to a career in web development. I have completed several courses through Codecademy to gain experience in HTML, CSS, and JavaScript. I am excited to bring my analytical skills to the field of web development!</p>
-          <a class="section-button grow" href="https://www.linkedin.com/in/andrew-smith-39838b219" target="_blank">Linkedin</a>
+          <a class="section-button change-color" href="https://www.linkedin.com/in/andrew-smith-39838b219" target="_blank">Linkedin</a>
         </section>
         <!-- PROJECTS -->
-        <section class="projects has-gutters" id="projects">
-          <h1>Projects.</h1>
+        <section class="projects" id="projects">
+          <h2>Projects.</h2>
           <div class="project-items-container">
-            <div class="project-item rotate-in-x" id="tea-cozy-project-container">
+            <div class="project-item rotate-in" id="tree-lafayette-project-container">

```

Comment:
I noticed that there is an ID set here, but it's not referenced in CSS or JS. I would recommend removing any classes or IDs on elements where they aren't referenced in other files.

Reply:
I think I started putting these on at the very beginning before I was sure if I would need an id for these elements. Now  realize that it is best practice to add classes and id's as needed rather than using them as labels. That is what comments are for!
Consider them removed. Thanks! :) 🥇